desktop/src/utils/codemod/expressionMethods.js

```diff
@@ -0,0 +1,202 @@
+import j from 'jscodeshift'
```

Comment:
Would be nice to have these codemod-related files on web instead of desktop.

I don't know all the pros & cons, but big pro: don't have to restart anything... can probably hot reload, too.